homeassistant/components/abode/__init__.py

```diff
@@ -239,9 +225,11 @@ def logout(event):
         _LOGGER.info("Logged out of Abode")
 
     if not hass.data[DOMAIN].polling:
-        hass.bus.listen_once(EVENT_HOMEASSISTANT_START, startup)
+        await hass.async_add_executor_job(hass.data[DOMAIN].abode.events.start)
 
-    hass.bus.listen_once(EVENT_HOMEASSISTANT_STOP, logout)
+    hass.data[DOMAIN].logout_listener = hass.bus.async_listen_once(
```

Comment:
We should initialize the `logout_listener` attribute in the `AbodeSystem` class. Set the attribute to  `None`.